src/Features/Core/Portable/CodeFixes/Suppression/WrapperCodeFixProvider.cs

```diff
@@ -12,10 +12,10 @@ internal sealed class WrapperCodeFixProvider : CodeFixProvider
         private readonly ImmutableArray<string> _originalDiagnosticIds;
         private readonly ISuppressionFixProvider _suppressionFixProvider;
 
-        public WrapperCodeFixProvider(ISuppressionFixProvider suppressionFixProvider, ImmutableArray<Diagnostic> originalDiagnostics)
+        public WrapperCodeFixProvider(ISuppressionFixProvider suppressionFixProvider, ImmutableArray<string> diagnosticIds)
```

Comment:
Will do that.


-          heejae

From: Manish Vasani [mailto:notifications@github.com]
Sent: Friday, October 9, 2015 12:29 PM
To: dotnet/roslyn <roslyn@noreply.github.com>
Cc: HeeJae Chang <Heejae.Chang@microsoft.com>
Subject: Re: [roslyn] moved Diagnostic usage to DiagnosticData in workspace layer (#5828)


In src/Features/Core/Portable/CodeFixes/Suppression/WrapperCodeFixProvider.cs<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2froslyn%2fpull%2f5828%23discussion_r41668885&data=01%7c01%7cHeejae.Chang%40microsoft.com%7c5dce4a95642f4dd4fc3108d2d0dfdc49%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=xlGkZf2vghzs8Kb5rOEWFQlVtQdonpWjW2533j2cq%2bo%3d>:

> @@ -12,10 +12,10 @@ internal sealed class WrapperCodeFixProvider : CodeFixProvider

>          private readonly ImmutableArray<string> _originalDiagnosticIds;

>          private readonly ISuppressionFixProvider _suppressionFixProvider;

>

> -        public WrapperCodeFixProvider(ISuppressionFixProvider suppressionFixProvider, ImmutableArray<Diagnostic> originalDiagnostics)

> +        public WrapperCodeFixProvider(ISuppressionFixProvider suppressionFixProvider, ImmutableArray<string> diagnosticIds)

okay makes sense, probably you can leave the .Distinct() call inside the ctor as the caller really shouldn't need to worry about it?

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fdotnet%2froslyn%2fpull%2f5828%2ffiles%23r41668885&data=01%7c01%7cHeejae.Chang%40microsoft.com%7c5dce4a95642f4dd4fc3108d2d0dfdc49%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=k2x59fcWfS8QGS%2fxUhMAjsc73GjSUPZsRURRsHe76M0%3d>.